File: backend.api/python.py
import sys
import json
import ast

input_data = ast.literal_eval(sys.argv[1])

input_data['numers_send'] = [num + 1 for num in input_data['numers_send']]

output_data = input_data
output_data['text_returned'] = 'send to node'

print(json.dumps(output_data))

sys.stdout.flush()

File: backend.api/python.ts
const spawner = require('child_process').spawn;

const data_to_pass_in = {
    text_send: 'sent to python script',
    numers_send: [1,2,3,4,5],
    text_returned: undefined
}

console.log('data send to python script:', data_to_pass_in);

const python_process = spawner('python', ['./python.py', JSON.stringify(data_to_pass_in)]);


python_process.stdout.on('data', (data: JSON) => {
    console.log('data received from python script: ', JSON.parse(data.toString()));
});